lib/ionic_notification/concerns/ionic_notificable.rb

```diff
@@ -2,17 +2,44 @@ module IonicNotification
   module Concerns
     module IonicNotificable
       extend ActiveSupport::Concern
-      include IonicNotification::Logger
 
       included do
         serialize :device_tokens, Array
 
         def notify(options = {})
-          return missing_device_tokens_logger unless respond_to?(:device_tokens)
-          return no_device_tokens_logger(self) unless device_tokens.count > 0
-          notification = IonicNotification::Notification.new(options.merge!(tokens: device_tokens))
+          logger = self.class.new_logger
+          return logger.missing_device_tokens unless respond_to?(:device_tokens)
+          tokens = device_tokens.uniq
+          return logger.no_device_tokens(self) unless tokens.count > 0
+          notification = IonicNotification::Notification.new(options.merge!(tokens: tokens))
           notification.send
         end
+
+        def notify_at(schedule, options = {})
+          options.merge!(scheduled: schedule.to_time.to_i)
+          notify options
+        end
+
+        def notify_in(delay, options = {})
+          schedule = Time.now + delay
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.